haystack_experimental/core/pipeline/pipeline.py

```diff
@@ -99,7 +138,12 @@ def _run_component(
             return cast(Dict[Any, Any], component_output)
 
     def run(  # noqa: PLR0915, PLR0912
-        self, data: Dict[str, Any], include_outputs_from: Optional[Set[str]] = None
+        self,
+        data: Dict[str, Any],
+        include_outputs_from: Optional[Set[str]] = None,
+        breakpoints: Optional[Set[Tuple[str, Optional[int]]]] = None,
+        resume_state: Optional[Dict[str, Any]] = None,
+        debug_path: Optional[str] = None,
```

Comment:
I believe this should be updated to 
```suggestion
        debug_path: Optional[str, Path] = None,
```